packages/server/graphql/public/mutations/setMeetingMusic.ts

```diff
@@ -0,0 +1,25 @@
+import {SubscriptionChannel} from '../../../../client/types/constEnums'
+import publish from '../../../utils/publish'
+import {MutationResolvers} from '../resolverTypes'
+
+const setMeetingMusic: MutationResolvers['setMeetingMusic'] = async (
```

Comment:
-1 this should be limited to the facilitator or at least a meeting member. Permissions can go into permissions.ts.